src/app/components/pages/node-url.component.ts

```diff
@@ -0,0 +1,52 @@
+import { Subscription } from 'rxjs';
+import { Component, OnInit, OnDestroy } from '@angular/core';
+import { ActivatedRoute, Router } from '@angular/router';
+
+import { ApiService } from 'app/services/api/api.service';
+import { ExplorerService } from 'app/services/explorer/explorer.service';
+
+/**
+ * Page for setting the URL of the local node the app must use as backend. The new URL is saved
+ * in local storage, so it is used again the next time the app is openned using the same
+ * domain name. If the URL is the string "null", any previously saved URL is deleted. Just after
+ * saving the URL, the browser is redirected to the main page.
+ */
+@Component({
+  selector: 'app-node-url',
+  template: '',
+})
+export class NodeUrlComponent implements OnInit, OnDestroy {
+  // Subscriptions that will be cleaned when closing the page.
+  private navParamsSubscription: Subscription;
+
+  constructor(
+    private route: ActivatedRoute,
+    private router: Router,
+    private explorerService: ExplorerService,
+    private apiService: ApiService,
+  ) { }
+
+  ngOnInit() {
+    // Check the URL params to detect the URL of the local node.
+    this.navParamsSubscription = this.route.params.subscribe(params => {
+      let nodeUrl: string = params['url'];
+
+      // If the url is "null", convert the value to a real null.
+      if (nodeUrl.toUpperCase() === 'null'.toUpperCase()) {
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/skycoin/skycoin-explorer/pullRequest?prid=7163626)
```suggestion
      if (nodeUrl.toUpperCase() === "null".toUpperCase()) {
```